core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -901,6 +926,7 @@ class SparkContext(config: SparkConf) extends Logging {
    * @param path Directory to the input data files, the path can be comma separated paths as the
    *             list of inputs.
    * @param minPartitions A suggestion value of the minimal splitting number for input data.
+   * @return RDD containing `Tuple2` with path to the file and the corresponding file content
```

Comment:
Although these don't add a lot it's kind of nice to doc `@return` for completeness here. I might change the language in all similar docs a bit to something more like "return an RDD of tuples of file path and corresponding file content". That is there are many tuples.